### ISSUE TITLE
Add instructions/configurations for using/scaling postgrest on different clouds

### ISSUE THREAD

**Issue by wolfgangwalther** [begriffs/postgrest-docs#453]
<div type='discussions-op-text'>

<sup>Originally posted by **gautam1168** September  5, 2021</sup>
One difficulty I faced while trying to get people to use postgrest where I work was the absence of a clear instructions to deploy on AWS that would take care of scaling. Heroku instructions are pretty close to beanstalk I think. But it would still have been much easier if we could have seen things like:

1. Instructions for deploying on beanstalk and some mention of how much it can be expected to scale.
2. Ways to add a cache (perhaps redis) or instructions on how to make a master-slave architecture on different clouds for postgrest.
3. Perhaps an official terraform setup to deploy it on different cloud providers. It could even have different versions for different scaling requirements (like one with nginx, one which spins up a master slave architecture for the dbs).
4. Maybe an official way to version control the database schemas and functions. This is left up to the user completely right now, but maybe we can have a cookbook for this. For example it could be a seed project as a git repository that you can clone. You edit any schemas etc you may want in .sql files, and commit it. And if you do docker compose up it starts everything with the updated migrations or alternatively updates the deployed app in aws.

Basically, we should make it easier to use for people who don't want to write libraries using postgrest. </div>
